src/Process/Pool/Logger/Message/SerializableProcess/FromProcessModel/Builder.php

```diff
@@ -0,0 +1,47 @@
+<?php
+declare(strict_types=1);
+
+namespace Neighborhoods\Kojo\Process\Pool\Logger\Message\SerializableProcess\FromProcessModel;
+
+use Neighborhoods\Kojo\Process\Pool\Logger\Message\SerializableProcessInterface as MessageProcessInterface;
+use Neighborhoods\Kojo\ProcessInterface as ProcessModelInterface;
+
+class Builder implements BuilderInterface
+{
+    use Factory\AwareTrait;
+    use \Neighborhoods\Kojo\Process\Pool\Logger\Message\SerializableProcess\Factory\AwareTrait;
+    /** @var ProcessModelInterface */
+    protected $processModelInterface;
+
+    public function build() : MessageProcessInterface
+    {
+        $process = $this->getProcessPoolLoggerMessageProcessFactory()->create();
+        $process->setProcessId($this->getProcessModelInterface()->getProcessId());
```

Comment:
I might extract `$this->getProcessModelInterface()` for readability, but no big deal